### ISSUE TITLE
Feature Request: Reply to comments in Slack threads (just like Google Docs has now)

### ISSUE THREAD

**Issue by atopiler** [integrations/slack#660]
If google docs can do this, so can github!  Being able to reply to PR comments directly in slack threads would be absolutely game-changing.  If there's one thing the github team decides to come back and work on to improve this integration, this needs to be it.  

Here's how the Google Docs integration works - so seamless!
![image](https://storage.googleapis.com/drive-slack/comment_replies_intro.gif)

**Comment by Mpreyzner** [comment]
This would be great and wouldn't require so much context switching.

**Comment by r-tock** [comment]
Yes, any plans of doing this? This will significantly reduce the loss of conversation and context switch

**Comment by huehnerlady** [comment]
That would be amazing - even better, to have not just comments, but everything belonging to a pull request should be in a thread, because otherwise it is just very confusing and unfortunately not really useful for me...

**Comment by marissamarym** [comment]
Hi, if anyone's interested in this functionality for issues, my product Instantish has 2-way syncing of comments between GitHub and Slack. 

We're still in early access, so if you'd like to use it, you can sign up at itsinstantish.com or DM me on Twitter (@marissamary) if you have questions.

**Comment by marissamarym** [comment]
Hi, for those interested in this functionality for issues, my product Instantish has 2-way syncing of comments between GitHub and Slack.

You can sign up for the early access at [itsinstantish.com](https://itsinstantish.com), or feel free to DM me on Twitter if you have questions. This is what notifs look like:

<img width="700" alt="dogcentered2" src="https://user-images.githubusercontent.com/1459660/75935346-d221e080-5e33-11ea-89cb-083ffe08e2d0.png">
